src/styles/main.scss

```
@@ -12,3 +12,36 @@ body {
 
 @import "_header.scss";
 @import "_catalog.scss";
+
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  justify-content: center;
+  gap: 48px;
+
+  &__card {
+    margin-bottom: 0;
+  }
+
+  &__card:not(:nth-child(4n)) {
+    margin-right: 0;
+  }
+
+  @media (max-width: 1024px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
+    }
```

цей `&` тут непотрібний))

```suggestion
    grid-template-columns: repeat(3, 200px);
```